src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts

```diff
@@ -1319,14 +1314,14 @@ export class TerminalTaskSystem extends Disposable implements ITaskSystem {
 	}
 
 	private _reviveTerminals(): void {
```

Comment:
Revive has a specific meaning in terminal land; a terminal that gets its process re-created based on its previous shell launch config. Should we stick to "reconnect" here?